pkg/services/govmomi/clustermodules/provider.go

```diff
@@ -80,28 +79,26 @@ func (cm *provider) DeleteModule(ctx context.Context, moduleUUID string) error {
 }
 
 // DoesModuleExist checks whether a module with a given name exists with the passed clusterRef and moduleUUID.
-func (cm *provider) DoesModuleExist(ctx context.Context, moduleUUID string, clusterRef types.ManagedObjectReference) (bool, error) {
+func (cm *provider) DoesModuleExist(ctx context.Context, moduleUUID string, _ types.ManagedObjectReference) (bool, error) {
```

Comment:
Let's drop the clusterRef parameter entirely.

It's slightly more works but it also means we don't have to retrieve the clusterRef in the "regular" case on the callsite of this func

Reply:
done